package.json

```diff
@@ -28,7 +29,8 @@
     "eslint-plugin-react": "^7.1.0",
     "mocha": "^5.2.0",
     "postcss-discard-comments": "^4.0.0",
-    "postcss-normalize-whitespace": "^4.0.0"
+    "postcss-normalize-whitespace": "^4.0.0",
```

Comment:
It doesn't look like `postcss-normalize-whitespace` is used anywhere? Let's remove if not needed